bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/config/EnOceanBridgeConfig.java

```diff
@@ -12,11 +12,13 @@
  */
 package org.openhab.binding.enocean.internal.config;
 
+import org.openhab.core.config.core.Configuration;
+
 /**
  *
  * @author Daniel Weber - Initial contribution
  */
-public class EnOceanBridgeConfig {
+public class EnOceanBridgeConfig extends Configuration {
```

Comment:
What is the reason for inheriting from `Configuration`?

Reply:
My fault, tried something and forgot to remove it. 👍